e2e/deno-e2e/tests/deno.spec.ts

```diff
@@ -130,7 +130,7 @@ console.log('123');
       );
 
       await expect(async () => {
-        await runNxCommandAsync(`lint ${appName}`);
+        await runNxCommandAsync(`lint ${appName} --skip-nx-cache`);
```

Comment:
some reason this is a cache it, but using `--skip-nx-cache` cause nx to correct see the files changes so the tests works. unsure why, also not sure why it's only lint when this isn't needed for the `should test deno app|lib w/options` tests either?